style.css

```diff
@@ -429,15 +434,187 @@ contact me section
 }
 
 /*
-hover effects
+desktop version
 */
 
-button:hover {
-  color: #fff;
-  background-color: #6070ff;
+.nav-desktop-list {
+  display: none;
+  width: 100%;
+  list-style: none;
+  justify-content: space-between;
+}
+
+.nav-desktop-list a {
+  text-decoration: none;
+  color: currentColor;
+}
+
+/*
+media queries
+*/
+
+@media (min-width: 768px) {
+  /*
+  navigation bar
+  */
+  .nav-icon-menu {
+    display: none;
+  }
+
+  .nav-my-logo {
+    flex: 1 1 auto;
+    box-sizing: border-box;
+    font-size: 1.5rem;
+  }
+
+  .nav-desktop-list {
+    display: flex;
+    flex: 1 1 auto;
+    width: 100%;
+    box-sizing: border-box;
+  }
+
+  .nav-flex-container {
+    height: 72px;
+  }
+
+  /*
+  header and welcom text
+  */
+
+  .header-text {
+    border-radius: 0 0 0 100px;
+    background-image: url('./img/desktop/Header\ bgdesktop-header-background.svg');
+    background-size: 100%;
+    background-repeat: no-repeat;
+    padding-left: 18%;
+    padding-right: 18%;
+    padding-bottom: 10%;
+  }
+
+  /*
+  Job Cards Section
+  */
+
+  .cards-container {
+    padding-left: 10%;
+    padding-right: 10%;
+  }
+
+  .card-element {
+    flex-direction: row;
+    padding: 2%;
+  }
+
+  .card-reverse {
+    flex-direction: row-reverse;
+  }
+
+  .card-element img {
+    flex: 1;
+  }
+
+  .text-box {
+    flex: 1;
+    padding-left: 2%;
+    padding-right: 2%;
+    box-sizing: border-box;
+  }
+
+  .technology-tags {
+    width: 40%;
+  }
```

Comment:
- [ ] Kindly, add proper padding for each technology tag, in order to, match the Figma design.

Your design:
![image](https://user-images.githubusercontent.com/86069740/236354732-31801da4-6178-4898-a5eb-c5b230793d4c.png)

Figma design:
![image](https://user-images.githubusercontent.com/86069740/236354793-ac157f62-7063-471d-b58e-89032d45b257.png)